packages/xo-server/src/xapi-object-to-xo.js

```diff
@@ -147,7 +147,9 @@ const TRANSFORMS = {
 
   // -----------------------------------------------------------------
 
-  host(obj) {
+  host(obj, dependents) {
+    dependents[obj.metrics] = obj.$id
```

Comment:
I used `dependents` in order to update the host record after a metrics update. This is needed because the host power state is based on a metrics property `live`.